README.md

```diff
@@ -335,6 +335,21 @@ describe('my-elem', () => {
 
 The filename and location of the resulting image will be based on the suite names and test name.
 
+Sometimes you might be writing tests to verify a component's position within the whole viewport. For snapshots of the whole page, set up your components and pass `document` to the assertion:
```

Comment:
I don't love this, feel free to wordsmith it.  But I did use the right form of set up!

Reply:
I'm going to rewrite all this anyway to separate vdiff out into its own section, so can have another look then.